chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_unittest.cc

```diff
@@ -69,8 +77,10 @@ void RecentTabsSubMenuModelTest::VerifyModel(
     item_data.enabled = false;
   }
 
-  ::VerifyModel(model,
-                UNSAFE_TODO(base::make_span(v_data.begin(), v_data.size())));
+  // The first two commands are History and History Clusters, but we disable
+  // History Clusters and upstream won't show it, so we should skip one command.
+  ::VerifyModel(model, UNSAFE_TODO(base::make_span(std::next(v_data.begin()),
+                                                   v_data.end())));
```

Comment:
I don't want to slow down the release of this branch of course, but there is this in 132 https://github.com/brave/brave-core/pull/26068/commits/55dcf0a643e357a9d39a838aa42a9bb501a80b79

Reply:
Thanks @cdesouza-chromium. Updated for master and 1.74.x, and 1.73.x.